lib/messagebird/conversation_client.rb

```diff
@@ -5,13 +5,25 @@
 
 module MessageBird
   class ConversationClient < HttpClient
-    ENDPOINT  = 'https://conversations.messagebird.com/v1/'
+    attr_reader :endpoint
 
+    @endpoint  = 'https://conversations.messagebird.com/v1/'
+    WHATSAPP_SANDBOX_ENDPOINT = 'https://whatsapp-sandbox.messagebird.com/v1/'
+
+    def initialize(access_key, features=[])
+      super(access_key)
+      if features.include? Client::ENABLE_CONVERSATIONS_WHATSAPP_SANDBOX then
+        @endpoint = WHATSAPP_SANDBOX_ENDPOINT
```

Comment:
What happens to other (non-whatsapp) endpoints when this base URL is used?

Reply:
The WhatsApp Sandbox team explained it to me as proxy that sits between the consumer and the normal API endpoint. I will confirm with them.

Reply:
Response "At the moment, it is rejected. In the future, it should just pass the request without modification."

Reply:
I have changed the way this works now, but will add a comment in the code.